coderd/database/migrations/000400_add_task_display_name.down.sql

```diff
@@ -0,0 +1,87 @@
+-- Drop view first before removing the display_name column from tasks
+DROP VIEW IF EXISTS tasks_with_status;
+
+-- Remove display_name column from tasks
+ALTER TABLE tasks DROP COLUMN display_name;
+
+-- Recreate view without the display_name column.
+-- This restores the view to its previous state after removing display_name from tasks.
+CREATE VIEW
```

Comment:
It seems adding the view recreation is required on both migrations, because otherwise sqlc would not generate the Task structure with `DisplayName` on up. AFAIU, since our queries all use `tasks_with_status` view and not the `tasks` table, PostgreSQL doesn't automatically update the view definition. That is why we need to force PostgreSQL to re-expand the `tasks.*` wildcard and pick up the new column. Let me know if there is a cleaner way of doing this.

Reply:
We need to get this implemented in our sqlc fork: https://github.com/Emyrk/sqlc/pull/1